src/main/java/org/jabref/gui/maintable/MainTable.java

```diff
@@ -149,10 +151,10 @@ public MainTable(MainTableDataModel model,
 
         mainTablePreferences.getColumnPreferences().getColumnSortOrder().forEach(columnModel ->
                 this.getColumns().stream()
-                    .map(column -> (MainTableColumn<?>) column)
-                    .filter(column -> column.getModel().equals(columnModel))
-                    .findFirst()
-                    .ifPresent(column -> this.getSortOrder().add(column)));
+                        .map(column -> (MainTableColumn<?>) column)
+                        .filter(column -> column.getModel().equals(columnModel))
+                        .findFirst()
+                        .ifPresent(column -> this.getSortOrder().add(column)));
```

Comment:
Please undo all your whitespace changes and use our checkstyle config for your jabref workspace.